File: Domain/Activity.cs
using System;
using System.Collections.Generic;

namespace Domain
{
    public class Activity
    {
        public Guid Id { get; set; }
        public string Name { get; set; }
        public string PhoneNumber { get; set; }
        public string Address { get; set; }
        public DateTime Date { get; set; }
        public string Description { get; set; }
        public string NoOfAcres { get; set; }
        public string Amount  { get; set; }
        public string Status { get; set; }
        public virtual ICollection<UserActivity> UserActivities { get; set; }
        public virtual ICollection<Financial> Financials { get; set; }
    }
}

File: client-app/src/app/models/activity.ts
export interface IActivity 
{
    id:string,
    name:string,
    phoneNumber:string,
    status:string,
    description:string,
    address:string,
    date:Date,
    amount:string,
    noOfAcres:string
}

export interface IActivityFormValues extends Partial<IActivity> {
    time?:Date
}


export class ActivityFormValues implements IActivityFormValues {
    id?: string = undefined;
    name: string = '';
    address: string ='';
    description:string = '';
    date?: Date = undefined;
    time?: Date = undefined;
    phoneNumber: string ='';
    noOfAcres: string ='';
    amount: string = '';
    status: string = '';


    constructor(init? : IActivityFormValues) 
    {
        if(init && init.date) 
        {
            init.time = init.date
        }
        Object.assign(this,init);
    }
}